.github/workflows/CI.yml

```diff
@@ -114,6 +114,10 @@ jobs:
             done
           fi
 
+      - name: Remove NAGL if no RDKit
+        if: ${{ matrix.rdkit == false }}
+        run: micromamba remove --force openff-nagl-base --yes
+
```

Comment:
This is not elegant but it's also borderline unfeasible to set up `_NAGL_AVAILABLE` given that the import check creates an import loop. This works well enough.

Reply:
Great! Will move the discussion there